src/presets/account.cairo

```diff
@@ -46,7 +47,7 @@ mod AccountUpgradeable {
     }
 
     #[constructor]
-    fn constructor(ref self: ContractState, public_key: felt252) {
+    pub(crate) fn constructor(ref self: ContractState, public_key: felt252) {
```

Comment:
I think we can address this in the migration to foundry issues, so that we can refactor the tests accordingly to be as consistent as possible (note that the udc for example require the pub keyword in the event struct, because there's no component, and this should be probably addressed as well).